### PR TITLE
Add filter on trackers jump

### PR DESCRIPTION
**Pull request by S-Dafarra**

### CODE REVIEW

**Comment by S-Dafarra** [comment]
@traversaro @GiulioRomualdi friendly ping

File: src/devices/openxrheadset/PosePublisher.cpp [hunk 1]
```diff
@@ -11,6 +11,53 @@
...
+}
+
+bool PosePublisher::rotationJumped()
+{
+    if (!m_lastValidData.pose.rotationValid || !m_data.pose.rotationValid)
+    {
+        return false; //Also the last "valid" pose is not valid, or the input is already not valid
+    }
+
+    if (!m_lastValidData.velocity.angularValid)
```
Comment (traversaro) [inline src/devices/openxrheadset/PosePublisher.cpp:39]: I am a bit lost, who is setting these flags?

Comment (S-Dafarra) [reply src/devices/openxrheadset/PosePublisher.cpp:39]: It is ultimately provided by ``OpenXR`` itself: https://github.com/ami-iit/yarp-device-openxrheadset/blob/4ea162bc2e35ccedb795aa268ce67ed60e95fd8d/src/devices/openxrheadset/impl/OpenXrInterfaceImpl.cpp#L140-L156

See https://www.khronos.org/registry/OpenXR/specs/1.0/html/xrspec.html#xrLocateSpace

Comment (traversaro) [reply src/devices/openxrheadset/PosePublisher.cpp:39]: Thanks!

**Review by traversaro — APPROVED** [review]

**Comment by S-Dafarra** [comment]
Thanks @traversaro!

**Pull request merged**